### ISSUE TITLE
[N7 Account Settings] Create New Contact Method page

### ISSUE THREAD

**Issue by Beamanator** [Expensify/App#11574]
Tracking issue: https://github.com/Expensify/Expensify/issues/182122

Detailed section of doc here: https://docs.google.com/document/d/17akqrbC-w8FqEoDLvcLmrX6iw4uaVrT_xlwAl-IhEzE/edit#heading=h.75ovxjptjj80

Note: This can only be implemented after `AddContactMethod` command has been finished - see https://github.com/Expensify/Expensify/issues/231920

**Comment by Beamanator** [comment]
On hold for WhatsApp Quality

**Comment by Beamanator** [comment]
Still on hold for now, might pick this up next week as discussions from here (https://github.com/Expensify/App/issues/11795) are indicating this could be useful to implement sooner rather than later

**Comment by Beamanator** [comment]
Unassigning since k2 is not liking that i have so many issues assigned to me on hold, will get back to this once N7 Account Settings if off hold (waiting on #passwordless too probably)

**Comment by Beamanator** [comment]
Monthly is fine, still waiting for #passwordless to wrap up

**Comment by MelvinBot** [comment]
@Beamanator, this ***Monthly*** task hasn't been acted upon in 6 weeks; closing.

If you disagree, feel encouraged to reopen it -- but pick your least important issue to close instead.

**Comment by JmillsExpensify** [comment]
Still not done, though I imagine this one is our next priority once https://github.com/Expensify/App/issues/11575 is squared away.

**Comment by Beamanator** [comment]
Agreed 👍

**Comment by Beamanator** [comment]
Hey @cristipaval I just realized this issue exists and you're working on this :D If this is a dupe, feel free to close it but please let me know so I can update the main tracker 😅

**Comment by cristipaval** [comment]
This is a dupe of https://github.com/Expensify/App/issues/15318

**Comment by cristipaval** [comment]
Linked the correct one in the tracking issue. Closing this one.

**Comment by JmillsExpensify** [comment]
Great, thanks for closing the loop!
